circle.yml

```diff
@@ -17,18 +17,14 @@ checkout:
   post:
     - ./scripts/circle-ci/pull-all-images.sh:
         background: true
-    - ./scripts/circle-ci/stop-services.sh
-
-general:
-  artifacts:
-    - "build/reports/profile"
+    - ./scripts/circle-ci/stop-services.sh:
+        background: true
```

Comment:
`stop-services.sh` only takes 10s. They should stop in time. Stopping the services is only to reduce memory pressure in external CircleCI so it doesn't really matter if the tests have started.